### DISCUSSION TITLE
Conflicto de espacios con el header white

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#1123] [Proposals]
Hola, hemos localizado un conflicto que tiene que ver con el header con fondo blanco. Tal y como vemos en la imagen, el espaciado entre el title del header y el título de módulo es muy grande porque se suman los 24px bottom del header con los 32px top de la sección.
¿Se os ocurre alguna solución desde Mistica?
Añado a @Winde a esta discussion a ver si entre ambos podemos encontrar una solución

![MicrosoftTeams-image (271)](https://user-images.githubusercontent.com/95479574/224951935-efc44f8d-3328-44cd-8e1c-75907b64e67a.png)

**Comment by MontseTelefonica** [comment]
Añado un punto. Lo ideal sería que pudiéramos usar el titulo de sección en lugar del header, pero a día de hoy el título de sección no puede ser dinámico.

**Comment by AnaMontes11** [comment]
Hola @MontseTelefonica!

Desde Mística tenemos 2 posibles soluciones para este conflicto:
1. No usar el `Header` y usar un `Title2`
2. Usar el `Header` pero eliminando ese padding que comentas

**Reply by MontseTelefonica** [reply]
Gracias Ana! No se sería posible usar el componente Title en lugar del header en el CMS, ¿que opinas @Winde?
La otra opción de eliminar el padding. ¿Sería posible integrar en el CMS una opción para "quitar" el padding top a una sección de forma manual en el CMS?

**Reply by yceballost** [reply]
Creemos que ese padding debe seguir existiendo en el componente, creo que el problema está en que la lógica del CMS no entiende de inverse y la mezcla de secciones con ambas del mismo color `background` provoca este bug visual.

https://rebrand.ly/as1xpx1

Entiendo que este problema tb ocurriría aunque quitemos ese padding al header y se mezclen otro tipo de secciones

aquí un ejemplo si hubiese una sección seguida con background
https://rebrand.ly/02q6tll

**Reply by Winde** [reply]
No es un problema de secciones => No estamos añadiendo espacio adicional debido a introducir secciones.

Este seria el caso:

https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AFATjAbgSxgdwBUIIAbAZwAIALbWCgegD4AdAOyQAkYBDWNAGW4BPCAFcALmwoVsZAJKtMMNGRgBeYADNu5GAF8p1Hnw2djyiuOziS65iAXjlrGOIA0FTdgBWECkIoAWwBnnBJ7BkYDViY2JAAlGDIABwhWMmwlQREJFlZpJAAhCAAPCmTeKGxWAHMiZI0ARgA2PTzpAoJrW0bGAGUIQJhLbpgkei6bGF7DAuKyiqgq2vqm1vaOgpQSbjAYKlI%2BSNmKcfmNs9K88cSUtIys4TFxa9vU9MyYbOeL%2BfLK6p1CANYAtNonJCTHr9QbDKxTcZQ6YbOalf5LQGrUHrE5bHZ7A4kI6xfIdS4lC70c5xehve6fb65NggNwgcRUGBDMgIADaIAAshAcGRxNw0CAALqsvC0dnc%2BA8gDMzQADBK9EA

Los componentes que colocamos debajo del header tienen un titulo y añaden una separación.
Esa separación se añade independientemente de si estan debajo del header o no debajo del header

Si el header es inverse => esa separacion es necesaria
Si el header es blanco => esa separación parece "too much"

No podemos quitarle el espaciado al modulo con titulo, pq como veis en el playroom si pensais que es el mismo componente dos veces, el espaciado procede.

Además de este caso de arriba de playroom, también procedería el espaciado  si el header de mistica fuera inverse:
https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AFATjAbgSxgdwBUIIAbAZwAIALbWCgegD4AdAOyQAkYBDWNAGW4BPCAFcALmwoVsZAJKtMMNGRgBeYOLSiYAXynUefDZyPKK47OJLrmIBeOWsY4gDQUAZtgBWECkIoAWwBnnBI7BkZ9ViY2JAAlGDIABwhWMmwlQREJFlZpJAAhCAAPCmTeKGxWAHMiZI0ARgA2XTzpAoIrG0bGAGUIQJgLbpgkei7rGF6DAuKyiqgq2vqm1vaOgpQSbjAYKlI%2BSNmKcfmNs9K88cSUtIys4TFxa9vU9MyYbOeL%2BfLK6p1CANYAtNonJCTHr9QbDSxTcZQ6YbOalf5LQGrUHrE5bHZ7A4kI6xfIdS4lC70c5xehve6fb65NggVwgcRUGBDMgIADaIAAshAcGRxNw0CAALqsvC0dnc%2BA8gDMzQADBLdEA

**Reply by MontseTelefonica** [reply]
Entonces, ¿que solución vemos a esto? Al menos alguna de cara a la salida del Hub?

**Reply by yceballost** [reply]
No habéis respondido a la parte donde comento cuando se quiera unir un header con una sección de color, tendríamos el caso opuesto :P
https://rebrand.ly/02q6tll

Y sé que no es el caso que tenéis, pero es que Mística no puede tener soluciones especificas, hay que pensar en casos globales y es legítimo que un equipo mañana quiera usar un header blanco seguido de una sección de color.

Esto mismo nos ocurre con una navigationBar y una lista por ejemplo.

en el caso que un "body" empiece con una lista no deberíamos aplicar los 24px de top que se suele definir para el primer elemento
https://rebrand.ly/12yz9dv

sin embargo, cuando no es un elemento como una lista o un hero si debes meter esos 24px.
https://rebrand.ly/8makffd

**Reply by lauraocana** [reply]
Como CMS no entiende de inverse por el momento, tendremos que salir con este error de diseño. No parece haber más opción @MontseTelefonica :(

**Comment by aweell** [comment]
Cierro esta discussion ya que se ha hecho release de la property `noPaddingY` en el header que elimina los paddings en el eje vertical y resuelve el problema detallado aquí.

https://github.com/Telefonica/mistica-web/pull/822
